lib/classes.rb

```diff
@@ -0,0 +1,43 @@
+class Player
+  attr_reader :name
+
+  def initialize(name, character)
+    @name = name
+    @character = character
+  end
+end
+
+class Game
+  def initialize; end
+
+  def check_winner(plays, boards_for_win = [])
+    if boards_for_win == []
+      boards_for_win = [[1, 2, 3], [1, 4, 7], [1, 5, 9], [2, 5, 8], [3, 5, 7], [3, 6, 9], [4, 5, 6], [7, 8, 9]]
+    end
+
+    win = false
+    boards_for_win.each do |board|
+      first_char = plays[board[0] - 1]
+      win = [plays[board[0] - 1], plays[board[1] - 1], plays[board[2] - 1]] == [first_char, first_char, first_char]
+      break if win
+    end
+    win
+  end
+
+  def check_draw(plays, boards_for_win = [])
+    if boards_for_win == []
+      boards_for_win = [[1, 2, 3], [1, 4, 7], [1, 5, 9], [2, 5, 8], [3, 5, 7], [3, 6, 9], [4, 5, 6], [7, 8, 9]]
+    end
+
+    draw = true
+
+    boards_for_win.each do |board|
+      winner_board = [plays[board[0] - 1], plays[board[1] - 1], plays[board[2] - 1]]
+      puts "winner_board & ['X'] = #{winner_board & ['X']}"
+      puts "winner_board & ['O'] = #{winner_board & ['O']}"
+      draw = false if winner_board & ['X'] == [] || winner_board & ['O'] == []
```

Comment:
## Inline comments
- No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
  -  [ ] Please remove `puts` and return a string variable
  ```ruby
  def check_draw(plays, boards_for_win = [])
  ...
  board_string=''
  board_string+=your_board_string
  board_string
  end
  ```

Reply:
there is no any 'puts' or 'gets' statement on the lines selected.
neither in any other place out 'main.rb' file.